### DISCUSSION TITLE
Remove old order data from legacy tables after switching to HPOS.

### DISCUSSION THREAD

**Discussion by DaanG47** [woocommerce/woocommerce#40988] [WooCommerce HPOS] [answered]
Hi!
Nowhere in the documentation is there any mention of cleaning the database after transferring orders to HPOS. Our Woocommerce store has been running flawlessly for several weeks now with HPOS enabled and synchronisation disabled, and we are curious to know if a tool will become available that can clean up the old order data (from the lecacy WP tables) since it has already synchronised to the HPOS tables.

Thanks in advance!

**Comment by rrennick** [comment, marked as answer]
The issue for purging order meta from post_meta is https://github.com/woocommerce/woocommerce/issues/39656.
